src/pages/add/join-wallet/join-wallet.ts

```diff
@@ -126,7 +133,31 @@ export class JoinWalletPage {
     } else {
       this.joinForm.get('recoveryPhrase').setValidators(null);
     }
+    this.joinForm.controls['recoveryPhrase'].setValue(null);
     this.joinForm.controls['selectedSeed'].setValue(seed);
+    this.processInvitation(this.joinForm.value.invitationCode);
+  }
+
+  private setDerivationPath(network: string): void {
+    let path: string =
+      network == 'testnet'
+        ? this.derivationPathForTestnet
+        : this.derivationPathByDefault;
+    this.joinForm.controls['derivationPath'].setValue(path);
+  }
+
+  public processInvitation(invitation: string): void {
+    if (this.regex.test(invitation)) {
+      this.logger.info('Processing invitation code...');
+      let walletData;
+      try {
+        walletData = this.bwcProvider.parseSecret(invitation);
+        this.setDerivationPath(walletData.network);
+        this.logger.info('Correct invitation code for ' + walletData.network);
+      } catch (ex) {
+        this.logger.warn(ex);
```

Comment:
This output of "logger" is hard to identify in the "Log" section if has not a prefix.

Reply:
Fixed